example-code/app/package.json

```diff
@@ -28,7 +28,7 @@
   "devDependencies": {
     "@itwin/build-tools": "workspace:*",
     "@itwin/eslint-plugin": "5.0.0-dev.1",
-    "@itwin/oidc-signin-tool": "^4.3.7",
+    "@itwin/oidc-signin-tool": "4.4.0",
```

Comment:
put the `^` sign back into the version bumps in this file and all the other ones

Reply:
> put the `^` sign back into the version bumps in this file and all the other ones

fixed